src/Enemy/EnemyCap.cpp

```diff
@@ -0,0 +1,186 @@
+#include "Enemy/EnemyCap.h"
+#include <al/Library/Base/String.h>
+#include <al/Library/LiveActor/ActorActionFunction.h>
+#include <al/Library/LiveActor/ActorFlagFunction.h>
+#include <al/Library/LiveActor/ActorInitFunction.h>
+#include <al/Library/LiveActor/ActorModelFunction.h>
+#include <al/Library/LiveActor/ActorMovementFunction.h>
+#include <al/Library/LiveActor/ActorResourceFunction.h>
+#include <al/Library/LiveActor/LiveActorFunction.h>
+#include <al/Library/Math/MathAngleUtil.h>
+#include <al/Library/Movement/EnemyStateBlowDown.h>
+#include <al/Library/Nerve/NerveSetupUtil.h>
+#include <al/Library/Nerve/NerveUtil.h>
+#include <al/Library/Obj/PartsFunction.h>
+#include <al/Library/Placement/PlacementFunction.h>
+#include <al/Library/Yaml/ByamlUtil.h>
+
+namespace {
+NERVE_IMPL(EnemyCap, Wait);
+NERVE_IMPL(EnemyCap, BlowDown);
+
+struct {
+    NERVE_MAKE(EnemyCap, Wait);
+    NERVE_MAKE(EnemyCap, BlowDown);
+} NrvEnemyCap;
+
+}  // namespace
+
+EnemyCap* EnemyCap::createEnemyCap(const char* name) {
+    return new EnemyCap(name);
+}
+
+EnemyCap::EnemyCap(const char* name) : al::LiveActor(name) {}
+
+static al::EnemyStateBlowDownParam* fallbackStateBlowDownParam = new al::EnemyStateBlowDownParam();
+
+// NON_MATCHING: Unsure how the al::StringTmp here works
```

Comment:
`decomp.me`?

Reply:
Created one. https://decomp.me/scratch/Ujrvm

Reply:
Ignoring the lower score, this certainly is a better match: https://decomp.me/scratch/mQG89

Before I dig into it any further, why did you decide to go with `al::StringTmp` instead? Is there something I am missing?

Reply:
Do the required adjustments to get it to the latest decomp.me version, then leave it as that (with `M`ajor mismatch)